src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,88 @@
+REM
+REM This script sync to the latest WCF source, build and submit Helix lab runs.
+REM Arguments:
+REM %1: Lab scripts location
+REM %2: WCF local git source location
+REM %3: Response files location
+
+@echo off
+setlocal
+
+set BridgeHost=
+pushd %2
+pushd bin\wcf\tools\Bridge
+if [%ERRORLEVEL%]==[0] (
+    call Bridge.exe -stop
+    popd
+)
+
+#REM remove bin directory to ensure a rebuild
+call git clean -xdf
+if NOT [%ERRORLEVEL%]==[0] (
+    echo ERROR: An error occurred while cleanning Git.
+    exit /b
+  )
+
+REM sync
+git checkout master
+if NOT [%ERRORLEVEL%]==[0] (
+    echo ERROR: An error occurred while checking out master.
+    exit /b
+  )
+git pull
+if NOT [%ERRORLEVEL%]==[0] (
+    echo ERROR: An error occurred while syncing master.
+    exit /b
+  )
+call startbridge.cmd  /allowRemote /remoteAddresses:*
+if NOT [%ERRORLEVEL%]==[0] (
+    echo ERROR: An error occurred while starting the bridge.
+    exit /b
+  )
+REM replaceBuildNumber.ps1 will update build number for all response files.
+copy /y %1\* .
+copy /y %3\* .
```

Comment:
Copying * into our repo root is a little dangerous.  If there is a file not in our .gitignore, there could be difficulties with them appearing as new files that need to be added.  And if they have a file of the same name, it will overwrite what is in our repo (imagine if they had a build.cmd). I recommend we instead copy them into a separate folder (like bin\WCFTemp) and access them from there.  Then we know a clean.cmd will remove them, and there is no danger of an overwrite.